### ISSUE TITLE
Is it possible to make this shadow invisible?

### ISSUE THREAD

**Issue by harangsa79** [helix-toolkit/helix-toolkit#2241]
![스크린샷 2024-06-16 211119](https://github.com/helix-toolkit/helix-toolkit/assets/155698005/e335035c-03cb-4bdd-98f8-42e3cfc06a7b)

![스크린샷 2024-06-16 212206](https://github.com/helix-toolkit/helix-toolkit/assets/155698005/0abdf20d-162e-48f1-bbcb-9dd51c754629)

When CullMode is set to Back, this is also not satisfactory as the circle is partially visible.

As shown in the photo, a shadow is created on [Tube] depending on the camera angle.

If I use CullMode it goes away but the problem is
 If you set Fornt and Back, the part where the tube starts will not be visible as a complete circle, but only a part of it will be visible.

The current light position is bound to [LookDirection].

Is there any other way other than CullMode?
[Material] is [PhongMaterial].
The same thing happens if you set it to [DiffuseMaterial].

**Comment by holance** [comment]
I assume your tube mesh is semi transparent? You have to do two pass of the mesh rendering to get rid of this artifacts if you can't use cullmode=back.

Create two MeshGeometryModel3D for each tube, bind their geometry and materials to the same, and only alter their cullmode. Set cullmode=back on First MeshGeometryModel3D, then set cullmode=front on second MeshGeometryModel3D.

**Comment by harangsa79** [comment]
thank you for the reply.
I never thought of using a double tube.^^

As you said, it is a slightly transparent tube, but we confirmed through testing that even a non-transparent tube creates a shadow.
As you rotate, the shadow changes accordingly.

After much thought, the solution I came up with was to set CullMode=Back and set the Tube Cap to both the front and back, and the shadow disappeared, and the original shape that had partially disappeared remained the same.

However, when I set the cap, it felt like it was blocked, which was a little disappointing, but I am satisfied.

Thank you again.
